bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -85,7 +85,7 @@
               </ul>
             </div>
             {% if l10n_has_tag('firefox_new_other_platforms') %}
```

Comment:
I think some other strings on the page may need to stay behind the tag, but this specific instance of that tag can be removed.